go/cmd/vttablet/vttablet.go

```diff
@@ -138,5 +142,22 @@ func main() {
 		// to update our state, so closing it in OnClose()
 		ts.Close()
 	})
+
+	if *tabletBackupAndExit {
+		go func() {
+			delay, err := agent.Healthy()
+			for err != nil || delay > 0 {
+				time.Sleep(time.Second)
+				delay, err = agent.Healthy()
+			}
+
+			err = agent.Backup(context.Background(), 1, logutil.NewConsoleLogger(), false)
+			if err != nil {
+				log.Fatalf("Backup() failed: %v", err)
+			}
+			// The only way to stop servenv is to SIGUSR1 it
```

Comment:
Did SIGTERM not work?

Reply:
The signal eventually handled by the `proc` package and it will handle SIGTERM and SIGUSR1, but the documentation says SIGUSR1 is the sanctioned signal to terminate the process